main.go

```diff
@@ -50,9 +50,11 @@ func init() {
 func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
+	var namespace string
 	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
 		"Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager.")
+	flag.StringVar(&namespace, "namespace", "", "The namespace in which the manager controls and reconciles resources.")
```

Comment:
Maybe add "Leave blank for watch all resources."?

Reply:
done